docs/update-listener-error.rst

```diff
@@ -0,0 +1,6 @@
+.. _update-listener-error:
+  
+Update the listener of a Terminated HTTPS Load Balancer
```

Comment:
I don't think this requires its own document. It's fine to include this in the docs/README.

Agree with @Niklaus-xie on the suggested wording change.